plugins/irma-server/index.js

```diff
@@ -48,7 +48,7 @@ module.exports = class IrmaServer {
   }
 
   _serverStateChange(newState) {
-    if ( newState == 'CONNECTED' )
+    if ( newState === 'CONNECTED' )
```

Comment:
I know that many JS developers have a healthy dislike of "double equals", but it makes sense in some cases. For example cases where you compare strings and you're not interested in wether the string is a string literal or a string object:

```javascript
> "hello" === new String("hello")
< false
> "hello" == new String("hello")
< true
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness

Reply:
Ah I thought this just was a mistake, if you did this on purpose then it is no problem. I will change it back.

Reply:
Yes, I try to stick to the rule:

* Do I want to compare **object** equality? Then use triple equals

```javascript
if ( typeof window === undefined ) ...
```

* Do I want to compare **value** equality? Then use double equals

```javascript
if ( input == 'CONNECTED' ) ...
```

But that's a bit of a personal choice I guess 😉